src/plugins/orangehrmDashboardPlugin/config/routes.yaml

```diff
@@ -5,9 +5,19 @@ apiv2_dashboard_employees_on_leave:
   defaults:
     _api: OrangeHRM\Dashboard\Api\EmployeeOnLeaveAPI
 
+##########################
+#       Mock API
+##########################
+
+apiv2_dashboard_employees_subunit_distribution:
```

Comment:
TODO::remove

Reply:
removed in https://github.com/orangehrm/orangehrm/pull/1428